LessonSix/src/test/java/ArrayExTest.java

```diff
@@ -0,0 +1,17 @@
+import org.junit.Before;
+import org.junit.Test;
+
+public class ArrayExTest {
+
+    private Array arr;
+
+    @Before
+    public void startTest() {
+        arr = new Array();
+    }
+
+    @Test(expected = RuntimeException.class)
+    public void testAfterLast4Ex() {
+        Array.AfterLast4(new int[]{0, 3, 7, 5, 0, 0, 3, 2});
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.